src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,31 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 1px;
+  width: getWidth($cellWidth);
+  margin: auto;
+  padding: $padding;
+
+  $days:
+    "mon" 1,
+    "tue" 2,
+    "wed" 3,
+    "thu" 4,
+    "fri" 5,
+    "sat" 6,
+    "sun" 7;
+
+  @each $name, $value in $days {
+    &--start-day-#{$name}:first-child {
```

Comment:
```scss
    &--start-day-#{$name} :first-child {

```

There must be a space before :first-child , otherwise it will not look inside the selector but it will find the first div with a class .calendar--start-day-29 in the parent.. Try and see